tests/models/semantic_role_labeler_test.py

```diff
@@ -50,3 +54,25 @@ def test_tag_returns_distributions_per_token(self):
         # Predictions are a distribution.
         numpy.testing.assert_almost_equal(numpy.sum(output["class_probabilities"], -1),
                                           numpy.array([1, 1, 1, 1]))
+
+    def test_bio_tags_correctly_convert_to_conll_format(self):
+        bio_tags = ["B-ARG-1", "I-ARG-1", "O", "B-V", "B-ARGM-ADJ", "O"]
+        conll_tags = convert_bio_tags_to_conll_format(bio_tags)
+        assert conll_tags == ["(ARG-1*", "*)", "*", "(V*)", "(ARGM-ADJ*)", "*"]
+
+    def test_perl_eval_script_can_run_on_printed_conll_files(self):
+        bio_tags = ["B-ARG-1", "I-ARG-1", "O", "B-V", "B-ARGM-ADJ", "O"]
+        sentence = ["Mark", "and", "Matt", "were", "running", "fast", "."]
+
+        gold_file_path = self.TEST_DIR + "gold_conll_eval.txt"
+        prediction_file_path = self.TEST_DIR + "prediction_conll_eval.txt"
+        with open(gold_file_path, "a+") as gold_file, open(prediction_file_path, "a+") as prediction_file:
+            # Use the same bio tags as prediction vs gold to make it obvious by looking
```

Comment:
I feel like this would be more useful if we actually checked that we get the metric we expected.  Would that be too hard to make happen in this test?